AppDB/appscale/datastore/scripts/update_index.py

```diff
@@ -43,39 +34,34 @@ def prettify_index(index, initial_indent=3):
 
 def main():
   """ Updates a composite index after prompting the user. """
-  try:
-    opts, remainder = getopt.getopt(
-      sys.argv[1:], 't:a:', ['type=', 'app_id='])
-  except getopt.GetoptError:
-    usage()
-    sys.exit(1)
-
-  db_type = None
-  app_id = None
-  for opt, arg in opts:
-    if opt in ('-t', '--type'):
-      db_type = arg
-    elif opt in ('-a', '--app_id'):
-      app_id = arg
-
-  if not db_type or not app_id:
-    usage()
-    sys.exit(1)
+  parser = argparse.ArgumentParser(description='Updates composite indexes')
+  parser.add_argument('--type', '-t', default='cassandra',
+                      help='The datastore backend type')
+  parser.add_argument('--app_id', '-a', required=True, help='The project ID')
```

Comment:
if we're moving to use project id in most cases, should the arg be changed to project_id?

Reply:
It would make more sense. I just generally try to change as little as possible to fix a problem in each pull request.